src/app/utils/field-validation/validators/__tests__/attachment-validation.spec.ts

```diff
@@ -219,9 +224,9 @@ describe('Attachment validation V3', () => {
 
   describe('Required or optional', () => {
     it('should disallow submission with no attachment if it is required', () => {
-      const formField = generateDefaultField(BasicField.Attachment, {
+      const formField = generateDefaultFieldV3(BasicField.Attachment, {
         attachmentSize: AttachmentSize.OneMb,
-      })
+      }) as FormFieldDto
```

Comment:
Is this typecasting still required since `generateDefaultFieldV3` returns `FormFieldDto`

Reply:
I missed out the removal of this cast. Thanks for the catch!